tests/unit/EntityHeader.spec.ts

```diff
@@ -185,8 +211,10 @@ describe('Entity Header - AUTHORIZED TO CONTINUE OUT badge', () => {
       }
 
       // cleanup
-      store.state.stateFiling = null
+      store.state.currentDate = null
+      store.state.filingHistoryList.filings.pop()
```

Comment:
👍 This is correct. The reason you need it is that the store is the same for all the tests in this file.